src/lib/containers/HasAccess.tsx

```diff
@@ -321,13 +316,47 @@ export default class HasAccess extends React.Component<
     )
   }
 
+  renderOpenAccessIcon() {
+    return (
+      <ThemeContext.Consumer>
```

Comment:
In a function component, we could just do

```tsx
const theme = useTheme()

return <IconSvg
              options={{
                icon: 'accessOpen',
                color: theme.colors.success,
              }}
            />
```

So this won't typically be this much code

Reply:
The reason for not wrapping the entire app in theme context is to avoid conflicts?